src/contexts/types.ts

```diff
@@ -79,7 +79,7 @@ export type Place = {
   id?: string
   latitude: string | null
   longitude: string | null
-  name: string
+  name: Record<string, string>
```

Comment:
This type should be more strict, i.e.

`type TranslatedField = Record<Language, string>`

and then:

```typescript
{
  ...
  name: TranslatedField
  ...
}
```